File: web-development/level-1/helsinki-part-1-2/Python/part1/week03/ex58.py
#ex58 week3 part1
#Reccuring a words

#self def- funcs

#main
def main():

    word_list = []

    #ask for user input
    while True:
        typed_word = input('Type a word: ')
        if typed_word in word_list:
            print('you gave the word: \' ', typed_word, ' \' twice \n' )
            break
        else:
            word_list.append(typed_word)

    print('the words you entered')
    for i in word_list:
        print(i)
        
#main func ends here
main()

File: helsinki-mooc/java/ex58.java
import java.util.Scanner;
import java.util.ArrayList;

public class ex58 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);

    ArrayList<String> word_list = new ArrayList<String>();

    while(true) {
      System.out.print("Type a word: ");
      String word = input.nextLine();

      if (word_list.contains(word)) {
        System.out.println("You have the " + word + " twice");
        break;
      } else {
        word_list.add(word);
      }
    }
  }// entry point
}
